5/음양더하기.js

```diff
@@ -0,0 +1,15 @@
+// for문을 이용
+function solution(absolutes, signs) {
+  let answer = 0;
+  for (let i = 0; i < absolutes.length; i++) {
```

Comment:
```suggestion
  let answer = 0;
  
  for (let i = 0; i < absolutes.length; i++) {
```
변수 선언과 다음 구문간의 줄바꿈!!